src/primitives/transaction.h

```diff
@@ -319,22 +326,30 @@ class CTransaction
 public:
     typedef boost::array<unsigned char, 64> joinsplit_sig_t;
 
-    // Transactions that include a list of JoinSplits are version 2.
-    static const int32_t MIN_CURRENT_VERSION = 1;
-    static const int32_t MAX_CURRENT_VERSION = 2;
+    // Transactions that include a list of JoinSplits are >= version 2.
+    static const int32_t SPROUT_MIN_CURRENT_VERSION = 1;
+    static const int32_t SPROUT_MAX_CURRENT_VERSION = 2;
+    static const int32_t OVERWINTER_MIN_CURRENT_VERSION = 3;
+    static const int32_t OVERWINTER_MAX_CURRENT_VERSION = 3;
 
-    static_assert(MIN_CURRENT_VERSION >= MIN_TX_VERSION,
+    static_assert(SPROUT_MIN_CURRENT_VERSION >= MIN_TX_VERSION,
+                  "standard rule for tx version should be consistent with network rule");
+
+    static_assert(OVERWINTER_MIN_CURRENT_VERSION >= MIN_TX_VERSION,
```

Comment:
This is misleading, because once Overwinter activates, `MIN_TX_VERSION` is no longer the minimum version the network rule allows. This was my motivation for having `OVERWINTER_[MIN|MAX]_TX_VERSION` defined in the consensus rules.

Reply:
I'm adding a constant to consensus.h